fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/ExistsPredicateOperator.java

```diff
@@ -8,21 +8,34 @@
 
 public class ExistsPredicateOperator extends PredicateOperator {
     private final boolean isNotExists;
+    private final boolean isSubquery;
```

Comment:
```
    //exists_predicate ::= KW_EXISTS subquery 
```

for ExistsPredicate, when it's child isn't subquery?

Reply:
It is unnecessary, I'll remove it.